src/nomad_parser_vasp/parsers/xml_parser.py

```diff
@@ -23,83 +23,88 @@
 # TODO move to schema parser and prepare all relevant annotations for every file format
 
 # ! vasprun.xml has many meta fields, explaining field semantics
-Program.name.m_annotations = dict(
-    xml=MappingAnnotationModel(path='modeling.generator.i[@name="program"]')
+Simulation.m_annotations = dict(xml=MappingAnnotationModel(path='modeling'))
+Simulation.program.name.m_annotations = dict(
+    xml=MappingAnnotationModel(path='modeling.generator.i[?"@name"="program"]')
 )
-Program.version.m_annotations = dict(
-    xml=MappingAnnotationModel(path='modeling.generator.i[@name="version"]')
+Simulation.program.version.m_annotations = dict(
+    xml=MappingAnnotationModel(path='modeling.generator.i[?"@name"="version"]')
 )
 # ? compilation mode?
-Program.compilation_host.m_annotations = dict(
-    xml=MappingAnnotationModel(path='modeling.generator.i[@name="platform"]')
+Simulation.program.compilation_host.m_annotations = dict(
+    xml=MappingAnnotationModel(path='modeling.generator.i[?"@name"="platform"]')
```

Comment:
also here. this should also work btw but a little slower since jmespath will do a global search